README.md

```diff
@@ -34,11 +34,13 @@ We enforce our coding standards with automated tooling ("linters"), that are usu
 
 Refer to [Mofo Style](https://github.com/MozillaFoundation/mofo-style) for more info.
 
-## Git Flow
+## GitHub Flow
 
-Many of our repos follow the Git Flow convention. If the default branch is `develop`, it is likely that this is the case.
+Active projects and future projects will use GitHub flow for branching. This means that new features should be written in feature branches based on `master`. When a feature is **complete**, open a pull request against `master` to have the new feature (or fix) brought into the base branch.
```

Comment:
we don't want to give up early review, though - catching methodology problems early saves a lot of time fixing later, so it's a good idea to mention that we're not "forbidding" Work In Progress PRs.